sorc/link_fv3gfs.sh

```diff
@@ -91,41 +91,33 @@ cd ${pwd}/../util               ||exit 8
         $LINK ../sorc/ufs_utils.fd/util/$file .
     done
 
+#-----------------------------------
+#--add gfs_wafs link if checked out
+if [ -d ${pwd}/gfs_wafs.fd ]; then
+#-----------------------------------
+ cd ${pwd}/../jobs               ||exit 8
+     $LINK ../sorc/gfs_wafs.fd/jobs/*                         .
+ cd ${pwd}/../parm               ||exit 8
+     [[ -d wafs ]] && rm -rf wafs
+    $LINK ../sorc/gfs_wafs.fd/parm/wafs                      wafs
+ cd ${pwd}/../scripts            ||exit 8
+    $LINK ../sorc/gfs_wafs.fd/scripts/*                      .
+ cd ${pwd}/../ush                ||exit 8
+    $LINK ../sorc/gfs_wafs.fd/ush/*                          .
+ cd ${pwd}/../fix                ||exit 8
+    [[ -d wafs ]] && rm -rf wafs
+    $LINK ../sorc/gfs_wafs.fd/fix/*                          .
+fi
```

Comment:
This WAFS block shouldn't be getting added. Need to recheck the sync with the `release/gfs.v16.3.21` branch and/or remove this block in your branch. The `release/gfs.v16.4` branch will need to get synced with the prior release branches again anyway but let's remove this block now.